src/components/modals/AddAccounts/steps/04-step-finish.js

```diff
@@ -30,6 +31,9 @@ const Text = styled(Box).attrs({
 function StepFinish({ currency, t, checkedAccountsIds }: StepProps) {
   return (
     <Box align="center" py={6}>
+      <RefreshAccountsOrdering onMount />
+      {/* the unmount case is useful to trigger a second refresh because most of the time countervalues were not yet loaded... */}
+      <RefreshAccountsOrdering onUnmount />
```

Comment:
what about having `onMount` and `onUnmount` on the same `RefreshAccountsOrdering`

Reply:
aha yeah 😅